### PR TITLE
enh(mc): refresh warp10 procedure

### PR DESCRIPTION
**Pull request by omercier**
## Description

Refresh Warp10

## Target version (i.e. version that this PR changes)

- [ ] 22.04.x
- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 1]
```diff
@@ -2,46 +2,506 @@
...
+
+1. Si vous avez utilisé un modèle d'hôte et coché la case **Créer aussi les services liés aux modèles**, les services associés au modèle ont été créés automatiquement, avec les modèles de services correspondants. Sinon, [créez les services désirés manuellement](/docs/monitoring/basic-objects/services) et appliquez-leur un modèle de service.
+2. Renseignez les macros désirées (par exemple, ajustez les seuils d'alerte). Les macros indiquées ci-dessous comme requises (**Obligatoire**) doivent être renseignées.
+
+<Tabs groupId="sync">
+<TabItem value="Fetch-Statistics" label="Fetch-Statistics">
+
+| Macro                        | Description                                                                                        | Valeur par défaut | Obligatoire |
+|:-----------------------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERAPPNAME                | Filter app name (can be a regexp)                                                                  |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:192]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 2]
```diff
@@ -2,46 +2,506 @@
...
+| --plink-path            | plink command path (default: none)                                                                                                                                     |
+| --plink-option          | Specify plink options (example: --plink-option='-T').                                                                                                                  |
+| --libssh-strict-connect | Connection won't be OK even if there is a problem (server known changed or server found other) with the ssh server.                                                    |
+| --filter-name           | Filter app name (can be a regexp).                                                                                                                                     |
+| --filter-counters       | Only display some counters (regexp can be used). Example: --filter-counters='calls'                                                                                    |
+| --warning-*-count       | /persecondWarning threshold. Can be: 'calls', 'bytes-values', 'bytes-keys'.                                                                                            |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:482]:
```suggestion
| --warning-*-count       | /persecond Warning threshold. Can be: 'calls', 'bytes-values', 'bytes-keys'.                                                                                            |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 3]
```diff
@@ -2,46 +2,506 @@
...
+| --critical-*-count      | /persecondCritical threshold. Can be: 'calls', 'bytes-values', 'bytes-keys'.                                                                                           |
+
+</TabItem>
+<TabItem value="Script-Statistics" label="Script-Statistics">
+
+| Option             | Description                                                                                                                            |
+|:-------------------|:---------------------------------------------------------------------------------------------------------------------------------------|
+| --filter-name      | Filter function name (can be a regexp).                                                                                                |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:490]: unclear

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 4]
```diff
@@ -2,46 +2,506 @@
...
+| --filter-counters  | Only display some counters (regexp can be used). Example: --filter-counters='^time$\|uses'                                             |
+| --warning-*-count  | /persecondWarning threshold. Can be: 'time-total' (delta), 'requests', 'ops', 'errors', 'bootstrap-loads', 'time' (delta), 'uses'.     |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:492]:
```suggestion
| --warning-*-count  | /persecond Warning threshold. Can be: 'time-total' (delta), 'requests', 'ops', 'errors', 'bootstrap-loads', 'time' (delta), 'uses'.     |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 5]
```diff
@@ -2,46 +2,506 @@
...
+| --critical-*-count | /persecondCritical threshold. Can be: 'time-total' (delta), 'requests', 'ops', 'errors', 'bootstrap-loads', 'time' (delta), 'uses'.    |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:493]:
```suggestion
| --critical-*-count | /persecond Critical threshold. Can be: 'time-total' (delta), 'requests', 'ops', 'errors', 'bootstrap-loads', 'time' (delta), 'uses'.    |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 6]
```diff
@@ -2,46 +2,504 @@
...
 
-The following services will be created:
+<Tabs groupId="sync">
+<TabItem value="Fetch-Statistics" label="Fetch-Statistics">
 
-  - Fetch-Statistics
-  - Script-Statistics
+| Macro                        | Description                                                                                        | Default value     | Mandatory   |
+|:-----------------------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERAPPNAME                | Filter app name (can be a regexp)                                                                  |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:192]: unclear

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 7]
```diff
@@ -2,46 +2,504 @@
...
+| --sshcli-option         | Specify ssh cli options (example: --sshcli-option='-o=StrictHostKeyChecking=no').                                                                                      |
+| --plink-command         | plink command (default: 'plink').                                                                                                                                      |
+| --plink-path            | plink command path (default: none)                                                                                                                                     |
+| --plink-option          | Specify plink options (example: --plink-option='-T').                                                                                                                  |
+| --libssh-strict-connect | Connection won't be OK even if there is a problem (server known changed or server found other) with the ssh server.                                                    |
+| --filter-name           | Filter app name (can be a regexp).                                                                                                                                     |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:478]: unclear

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 8]
```diff
@@ -2,46 +2,504 @@
...
+| --filter-perfdata                          | Filter perfdata that match the regexp. Example: adding --filter-perfdata='avg' will remove all metrics that do not contain 'avg' from performance data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:375]:
```suggestion
| --filter-perfdata                          | Keep only perfdata that match the regexp. Example: adding --filter-perfdata='avg' will remove all metrics that do not contain 'avg' from performance data.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 9]
```diff
@@ -2,46 +2,504 @@
...
+| --source-encoding                          | Define the character encoding of the response sent by the monitored resource Default: 'UTF-8'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:394]:
```suggestion
| --source-encoding                          | Define the character encoding of the response sent by the monitored resource. Default: 'UTF-8'.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 10]
```diff
@@ -2,46 +2,504 @@
...
+#### Custom modes options
+
+All **custom modes** specific options are listed here:
+
+<Tabs groupId="sync">
+<TabItem value="file" label="file">
+
+| Option                 | Description                                                                                                                                                                                                                                   |
+|:-----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --hostname             | Hostname to query (with ssh).                                                                                                                                                                                                                 |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:405]:
```suggestion
| --hostname             | Hostname of the resource to query (with SSH).                                                                                                                                                                                                                 |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 11]
```diff
@@ -2,46 +2,504 @@
...
+| --command              | Command to get information (default: 'cat').                                                                                                                                                                                                  |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:406]: ?

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 12]
```diff
@@ -2,46 +2,504 @@
...
+| --statefile-cipher     | Define the cipher algorithm to encrypt the cache (default: 'AES').                                                                                                                                                                            |
+
+</TabItem>
+<TabItem value="web" label="web">
+
+| Option                 | Description                                                                                                                                                                                                                                   |
+|:-----------------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| --hostname             | Endpoint hostname.                                                                                                                                                                                                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:427]: ?

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 13]
```diff
@@ -2,46 +2,504 @@
...
+| --port                 | Port used (default: 80)                                                                                                                                                                                                                       |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:428]:
```suggestion
| --port                 | Port of the resource to connect to (default: 80).                                                                                                                                                                                                                      |
```

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 14]
```diff
@@ -2,46 +2,504 @@
...
-| Field            | Value                       |
-| :--------------- | :-------------------------- |
-| Name             | *Name of the host*          |
-| Alias            | *Description*               |
-| IP Address / DNS | *Can be localhost*          |
-| Monitored from   | *Poller used to monitor*    |
-| Templates        | App-DB-Warp10-Sensision-Web |
+| Macro                       | Description                                                                                          | Default value     | Mandatory   |
+|:----------------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARP10SENSISIONUSERNAME     | Endpoint username                                                                                    |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:173]: ?

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 15]
```diff
@@ -2,46 +2,504 @@
...
+| --port                 | Port used (default: 80)                                                                                                                                                                                                                       |
+| --proto                | Specify https if needed (default: 'http')                                                                                                                                                                                                     |
+| --urlpath              | URL to scrape metrics from (default: '/metrics').                                                                                                                                                                                             |
+| --username             | Endpoint username.                                                                                                                                                                                                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:431]: ?

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 16]
```diff
@@ -2,46 +2,504 @@
...
+| --password             | Endpoint password.                                                                                                                                                                                                                            |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:432]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 17]
```diff
@@ -2,46 +2,506 @@
...
+
+1. Ajoutez un hôte à Centreon depuis la page **Configuration > Hôtes**.
+2. Complétez les champs **Nom**, **Alias** & **IP Address/DNS** correspondant à votre ressource.
+3. Appliquez le modèle d'hôte **App-DB-Warp10-Sensision-Web-custom**. Une liste de macros apparaît. Les macros vous permettent de définir comment le connecteur se connectera à la ressource, ainsi que de personnaliser le comportement du connecteur.
+4. Renseignez les macros désirées. Attention, certaines macros sont obligatoires, notamment la macro permettant de définir le [custom mode](#custom-modesdisponibles), c'est-à-dire la méthode de connexion à la ressource.
+
+| Macro                       | Description                                                                                          | Valeur par défaut | Obligatoire |
+|:----------------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARP10SENSISIONUSERNAME     | Endpoint username                                                                                    |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:173]: ?

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 18]
```diff
@@ -2,46 +2,506 @@
...
+| WARP10SENSISIONPASSWORD     | Endpoint password                                                                                    |                   |             |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:174]: ?

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 19]
```diff
@@ -2,46 +2,504 @@
...
-| :--------------- | :-------------------------- |
-| Name             | *Name of the host*          |
-| Alias            | *Description*               |
-| IP Address / DNS | *Can be localhost*          |
-| Monitored from   | *Poller used to monitor*    |
-| Templates        | App-DB-Warp10-Sensision-Web |
+| Macro                       | Description                                                                                          | Default value     | Mandatory   |
+|:----------------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARP10SENSISIONUSERNAME     | Endpoint username                                                                                    |                   |             |
+| WARP10SENSISIONPASSWORD     | Endpoint password                                                                                    |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:174]: ?

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 20]
```diff
@@ -2,46 +2,504 @@
...
+| WARNINGDATAPOINTSPERSECOND   | Thresholds.                                                                                        |                   |             |
+| CRITICALDATAPOINTSPERSECOND  | Thresholds.                                                                                        |                   |             |
+| EXTRAOPTIONS                 | Any extra option you may want to add to the command (a --verbose flag for example). All options are listed [here](#available-options). | --verbose         |             |
+
+</TabItem>
+<TabItem value="Script-Statistics" label="Script-Statistics">
+
+| Macro                           | Description                                                                                        | Default value     | Mandatory   |
+|:--------------------------------|:---------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| FILTERAPPNAME                   | Filter function name (can be a regexp)                                                             |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:216]: unclear

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 21]
```diff
@@ -2,46 +2,504 @@
...
+| --filter-counters       | Only display some counters (regexp can be used). Example: --filter-counters='calls'                                                                                    |
+| --warning-*-count       | /persecondWarning threshold. Can be: 'calls', 'bytes-values', 'bytes-keys'.                                                                                            |
+| --critical-*-count      | /persecondCritical threshold. Can be: 'calls', 'bytes-values', 'bytes-keys'.                                                                                           |
+
+</TabItem>
+<TabItem value="Script-Statistics" label="Script-Statistics">
+
+| Option             | Description                                                                                                                            |
+|:-------------------|:---------------------------------------------------------------------------------------------------------------------------------------|
+| --filter-name      | Filter function name (can be a regexp).                                                                                                |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:488]: unclear

File: pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 22]
```diff
@@ -2,46 +2,504 @@
...
+| --ssh-backend           | Define the backend you want to use. It can be: sshcli (default), plink and libssh.                                                                                     |
+| --ssh-username          | Define the user name to log in to the host.                                                                                                                            |
+| --ssh-password          | Define the password associated with the user name. Cannot be used with the sshcli backend. Warning: using a password is not recommended. Use --ssh-priv-key instead.   |
+| --ssh-port              | Define the TCP port on which SSH is listening.                                                                                                                         |
+| --ssh-priv-key          | Define the private key file to use for user authentication.                                                                                                            |
+| --sshcli-command        | ssh command (default: 'ssh').                                                                                                                                          |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:471]:
```suggestion
| --sshcli-command        | SSH command (default: 'ssh').                                                                                                                                          |
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md [hunk 23]
```diff
@@ -2,46 +2,506 @@
...
+| --sshcli-path           | ssh command path (default: none)                                                                                                                                       |
+| --sshcli-option         | Specify ssh cli options (example: --sshcli-option='-o=StrictHostKeyChecking=no').                                                                                      |
+| --plink-command         | plink command (default: 'plink').                                                                                                                                      |
+| --plink-path            | plink command path (default: none)                                                                                                                                     |
+| --plink-option          | Specify plink options (example: --plink-option='-T').                                                                                                                  |
+| --libssh-strict-connect | Connection won't be OK even if there is a problem (server known changed or server found other) with the ssh server.                                                    |
```
[the remark this answers is not in the document]
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:479]: I have no idea...

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md (hunk 2 above)
Comment (omercier) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-warp10-sensision.md:482]: I think there's a parsing issue here. we should remove /persecond

**Pull request merged**